docs/wmco-prerequisites.md

```diff
@@ -0,0 +1,51 @@
+| Azure                    | 4.6+                        | WMCO 1.0+                |
+| VMware vSphere           | 4.7+                        | WMCO 2.0+                |
+
+## Supported Windows Server versions
+The following table outlines the supported
+[Windows Server version](https://docs.microsoft.com/en-us/windows/release-health/release-information) based on the 
+applicable cloud provider.
+
+Note: Any unlisted Windows Server version are NOT supported, and will cause errors. To prevent 
+these errors, only use the appropriate version according to the cloud provider in use. 
+
+| Cloud Provider      | Supported Windows Server version                                        |
+| -----------         | -----------                                                             |
+| AWS                 | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809  |
+| Azure               | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809  |
+| VMware vSphere      | Windows Server Semi-Annual Channel (SAC): Windows Server 2004           |
+
+## Supported Networking
+[OVNKubernetes hybrid networking](setup-hybrid-OVNKubernetes-cluster.md) is the only supported networking configuration.
+The following tables outline the type of networking configuration and Windows Server versions to be used based on your 
+cloud provider. This must be completed during the installation of the cluster.
+  
+Note: OpenShiftSDN networking is the default network for OpenShift clusters. It is NOT supported by WMCO.
+
+| Cloud Provider      | Supported networking                                                                          |
+| -----------         | -----------                                                                                   | 
+| AWS                 | Hybrid OVNKubernetes                                                                          |
+| Azure               | Hybrid OVNKubernetes                                                                          |
+| VMware vSphere      | Hybrid OVNKubernetes with a [Custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vsphere)|
+
+| Hybrid OVNKubernetes      | Supported Windows Server version                                      |
+| -----------               | -----------                                                           |
+| Default VXLAN port        | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809|
+| Custom VXLAN port         | Windows Server Semi-Annual Channel (SAC): Windows Server 2004         |
+
+## Supported Installation method
+* Installer-Provisioned Infrastructure installation method is the only supported installation method. This is 
+consistent across all supported cloud providers.
+  
+* User-Provisioned Infrastructure is NOT supported.
+
+# vSphere Specific Requirements
+[VMware vSphere pre-requisites](vsphere-prerequisites.md)
```

Comment:
Another nit: perhaps make this line a sentence `Please refer to [VMware vSphere pre-requisites](vsphere-prerequisites.md)` instead of just dropping the link